profiles/package.mask

```diff
@@ -33,6 +33,11 @@
 
 #--- END OF EXAMPLES ---
 
+# Aliaksei Urbanski <aliaksei.urbanski@gmail.com> (2024-06-24)
+# Js2Py is vulnerable to a sandbox escape (CVE-2024-28397)
+# Please remove it or upgrade to the latest revision!
+<dev-python/js2py-0.74-r1
```

Comment:
you can't "just mask it" since <0.74 is stable it breaks reverse dependencies

after this is merged... if it will be I will file a stablereq